pkg/controller/compliancescan/forwarder.go

```diff
@@ -0,0 +1,46 @@
+package compliancescan
+
+import (
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	compv1alpha1 "github.com/ComplianceAsCode/compliance-operator/pkg/apis/compliance/v1alpha1"
+)
+
+func ForwarderFactory(s *compv1alpha1.ComplianceScan) Forwarder {
```

Comment:
nit: This could be New() or similar. They used to have functions named as factories in old Kube and it gave me Java-chills :)